### ISSUE TITLE
iqiyi Name or service not known

### ISSUE THREAD

**Issue by asuna** [soimort/you-get#486]
```bash
{ ~ }  » /bin/you-get http://http://www.iqiyi.com/v_19rro3snwg.html                             ~
Traceback (most recent call last):
  File "/bin/you-get", line 11, in <module>
    sys.exit(main())
  File "/usr/lib/python3.2/site-packages/you_get/__main__.py", line 91, in main
    main()
  File "/usr/lib/python3.2/site-packages/you_get/common.py", line 998, in main
    script_main('you-get', any_download, any_download_playlist)
  File "/usr/lib/python3.2/site-packages/you_get/common.py", line 892, in script_main
    download_main(download, download_playlist, args, playlist, output_dir=output_dir, merge=merge, info_only=info_only)
  File "/usr/lib/python3.2/site-packages/you_get/common.py", line 784, in download_main
    download(url, **kwargs)
  File "/usr/lib/python3.2/site-packages/you_get/common.py", line 990, in any_download
    m, url = url_to_module(url)
  File "/usr/lib/python3.2/site-packages/you_get/common.py", line 981, in url_to_module
    conn.request("HEAD", video_url)
  File "/usr/lib/python3.2/http/client.py", line 983, in request
    self._send_request(method, url, body, headers)
  File "/usr/lib/python3.2/http/client.py", line 1021, in _send_request
    self.endheaders(body)
  File "/usr/lib/python3.2/http/client.py", line 979, in endheaders
    self._send_output(message_body)
  File "/usr/lib/python3.2/http/client.py", line 824, in _send_output
    self.send(msg)
  File "/usr/lib/python3.2/http/client.py", line 762, in send
    self.connect()
  File "/usr/lib/python3.2/http/client.py", line 740, in connect
    self.timeout, self.source_address)
  File "/usr/lib/python3.2/socket.py", line 408, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
socket.gaierror: [Errno 8] Name or service not known
```
下载爱奇艺视频的时候报错，试了几个视频都不行

**Comment by jackyzy823** [comment]
there are two http:// in your links and due to the fix for #481 ,new error introduced. #481 should be considered as a special case.
it's my fault.feel sorry for that.

**Comment by asuna** [comment]
Sorry, i didn't find that two http:// ...... These days I always get 2 http:// when i copy url from my browser : (
